### PR TITLE
Add GetAllParams function

### PR DESCRIPTION
**Pull request by helo9**
This PRs aim to introduce a GetAllParams function, as already started in [MAVSDK-Proto PR#175](https://github.com/mavlink/MAVSDK-Proto/pull/175). 
The only testing I have done so far is the integration test I have added. Therefore I keep it a draft for now.

### CODE REVIEW

File: src/plugins/param/param_impl.cpp [hunk 1]
```diff
@@ -47,6 +47,31 @@ Param::Result ParamImpl::set_param_float(const std::string& name, float value)
...
+            tmp_param.value = parampair.second.get_float();
+            res.float_params.push_back(tmp_param);
+        } else if (parampair.second.is_int32()) {
+            Param::IntParam tmp_param;
+            tmp_param.name = parampair.first;
+            tmp_param.value = parampair.second.get_int32();
+            res.int_params.push_back(tmp_param);
+        } /*else {
+            LogDebug() << "missed " << parampair.first << " " << parampair.second.get_string();
+        }*/
```
Comment (JonasVautherin) [inline src/plugins/param/param_impl.cpp:69]:
```suggestion
        }
```

**Review by JonasVautherin — CHANGES_REQUESTED** [review]
Seems good to me! The docs CI test is failing, with:

```
Compound mavsdk::Param::IntParam is not documented.
Compound mavsdk::Param::AllParams is not documented.
Compound mavsdk::Param::FloatParam is not documented.
```

I think that's because the `messages` miss the documentation [here](https://github.com/mavlink/MAVSDK-Proto/blob/32a60e437e84daf1ec440504c374e7cd46256592/protos/param/param.proto#L80-L81), [here](https://github.com/mavlink/MAVSDK-Proto/blob/32a60e437e84daf1ec440504c374e7cd46256592/protos/param/param.proto#L85-L86) and [here](https://github.com/mavlink/MAVSDK-Proto/blob/32a60e437e84daf1ec440504c374e7cd46256592/protos/param/param.proto#L90-L91)

**Review by JonasVautherin — APPROVED** [review]
Thanks a lot!

**Comment by helo9** [comment]
> Thanks a lot!

Thank you for updating the protos :+1:

**Pull request merged**